stdlib/random.pyi

```diff
@@ -68,7 +68,7 @@ class Random(_random.Random):
     def __init__(self, x: Any = ...) -> None: ...
     # Using other `seed` types is deprecated since 3.9 and removed in 3.11
     if sys.version_info >= (3, 9):
-        def seed(self, a: int | float | str | bytes | bytearray | None = ..., version: int = ...) -> None: ...  # type: ignore[override]
+        def seed(self, a: float | str | bytes | bytearray | None = ..., version: int = ...) -> None: ...  # type: ignore[override]
```

Comment:
This one feels a little worse to me, any objections to noqa?

Reply:
No, feel free to revert and add a `noqa`!